shared/adw-data-lake/build-data-lake/introduction/introduction.md

```diff
@@ -0,0 +1,76 @@
+# Introduction
+
+## About this Workshop
+
+The labs in this workshop walk you through all the steps to build a Data Lake using Autonomous Warehouse (ADW), Oracle Object Storage, and optionally Oracle Cloud Infrastructure (OCI) Data Catalog. You will learn how to load (and link) data into your ADW instance using different from a variety of locations and in different formats, using Autonomous Database tools and APIs. You will analyze the linked and loaded data with SQL and also query Oracle Object Storage buckets and Amazon S3 (Simple Storage Service) from within ADW.
```

Comment:
I think "using different" should be removed from the second sentence (or something is missing after those words)

Reply:
I agree. Please remove "using different".